### ISSUE TITLE
Can't compile on Ubuntu with Vapor.

### ISSUE THREAD

**Issue by atjason** [JustHTTP/Just#54]
Hi,
I'm trying to use Just on Ubuntu with Vapor, but has compile error. Maybe it's platform related and can't be supported, I still want to report this issue to let you have sense.

Main steps:
- [Install Vapor on Ubuntu](https://vapor.github.io/documentation/getting-started/install-swift-3-ubuntu.html)
- [Install Toolinbox](https://vapor.github.io/documentation/getting-started/install-toolbox.html)
- [Create a Hello World project](https://vapor.github.io/documentation/getting-started/hello-world.html)
- Edit 'Package.swift' and add Just.
- Run `swift build`, will get these errors:

```
./Sources/Just/Just.swift:99:56: error: downcast from '[String : String]?' to '[String : String]' only unwraps optionals; did you mean to use '!'?
    return (self as? HTTPURLResponse)?.allHeaderFields as? [String: String]
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ ^   ~~~~~~~~~~~~~~~~
./Sources/Just/Just.swift:181:40: error: incorrect argument label in call (have 'withResponseHeaderFields:for:', expected 'withResponseHeaderFields:forURL:')
      foundCookies = HTTPCookie.cookies(withResponseHeaderFields: headers,
                                       ^
./Sources/Just/Just.swift:209:12: error: cannot convert value of type 'String' to type 'NSString' in coercion
          ($0 as NSString).trimmingCharacters(in: CharacterSet.whitespaces)
           ^~
./Sources/Just/Just.swift:954:21: error: use of unresolved identifier 'DispatchSemaphore'
    let semaphore = DispatchSemaphore(value: 0)
                    ^~~~~~~~~~~~~~~~~


./Sources/Just/Just.swift:994:39: error: use of unresolved identifier 'DispatchTime'
      let timeout = timeout.flatMap { DispatchTime.now() + $0 }
                                      ^~~~~~~~~~~~
./Sources/Just/Just.swift:995:12: error: use of unresolved identifier 'DispatchTime'
        ?? DispatchTime.distantFuture
           ^~~~~~~~~~~~
./Sources/Just/Just.swift:1095:22: error: cannot convert value of type 'NSMutableData' to type 'Data' in coercion
        data: config.data as Data,
              ~~~~~~~^~~~
```

Hope Just could support Ubuntu with Vapor. Thanks in advance.

**Comment by cipi1965** [comment]
Hey @atjason, it should be fixed in 0.6 :)

**Comment by atjason** [comment]
Get it, thanks :)
